ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/ip_name_server_groups.yml

```diff
@@ -0,0 +1,33 @@
+---
+ip_name_server_groups:
+  - name: mynameserver1
+    ip_domain_list: domain-list1
+    dns_domain: arista.avd.com
+    name_servers:
+      - ip_address: 8.8.8.8
+        vrf: vrf1
+        priority: 0
+      - ip_address: 1.1.1.1
+      - ip_address: 2.2.2.2
+        vrf: vrf1
+        priority: 1
+      - ip_address: 2.2.2.1
+        vrf: vrf1
+      - ip_address: 2.2.2.4
+        vrf: vrf1
+        priority: 4
+      - ip_address: 2.2.2.6
+        vrf: b_vrf
+        priority: 3
+      - ip_address: 2.2.2.7
+        vrf: a_vrf
+        priority: 3
+
+  - name: mynameserver0
+    ip_domain_list: domain-list1
```

Comment:
you can improve the coverage by not defining ip_domain_list and dns_domain here. Add one more name server when just define dns_domain and do not define name_servers.

Reply:
Fixed.

Reply:
Added.